app/assets/stylesheets/hoge/fuga.css.scss

```diff
@@ -0,0 +1,40 @@
+.cat{}
+
+.dog {
+  color: #ff00ff;
+}
+
+p {
+  background: #ab;
+}
+
+
+span.hoge {
+        padding: 10px;
+}
+
+.piyo {
+  margin: 0.5em;
```

Comment:
__[scss_lint]__
* [LeadingZero] `0.5` should be written without a leading zero as `.5`

<!-- Comment by [SideCI](https://sideci.com) -->